package.json

```diff
@@ -23,39 +26,69 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/smg-automotive/fe-example-pkg.git"
+    "url": "https://github.com/smg-automotive/components-pkg.git"
+  },
+  "homepage": "https://github.com/smg-automotive/components-pkg#readme",
+  "peerDependencies": {
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
   },
-  "homepage": "https://github.com/smg-automotive/fe-example-pkg#readme",
   "devDependencies": {
+    "@babel/core": "7.17.10",
+    "@react-theming/storybook-addon": "1.1.5",
     "@rollup/plugin-commonjs": "22.0.0",
     "@rollup/plugin-node-resolve": "13.3.0",
     "@rollup/plugin-typescript": "8.3.2",
+    "@smg-automotive/eslint-config": "1.0.0-initial-setup-b1ec39ecd70bc552555f6e904f29371e55ae31e6.1",
+    "@storybook/addon-actions": "6.5.0-beta.8",
+    "@storybook/addon-controls": "6.5.0-beta.8",
```

Comment:
My suggestion would be adding the `@storybook/addon-viewport` as well. I remember that helped me a lot to display the stories in different sizes and layouts. Otherwise, we need to do it with the devTool and that is not really helpful because it takes the whole window and it doesn't resize the element we are looking for. What do you think?

Reply:
It's actually already included in [essentials](https://storybook.js.org/addons/tag/essentials/): 
<img width="269" alt="Screenshot 2022-05-12 at 09 58 16" src="https://user-images.githubusercontent.com/144707/168021157-ed34fbd7-7dd2-4f81-917b-b9326b1712c0.png">
Let me clean those deps an remove the ones that we require through essentials